workflow-templates/deploy-gh-pages.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will do a clean install of node dependencies, build a demo project, and deploy it to Github Pages
+
+name: Deploy to Github Pages
+on:
+  push:
+    branches: [$default-branch]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+        with:
+          persist-credentials: false 
+
+      - name: Install and Build 🔧
+        run: |
+          npm install
+          npm run-script build-demo
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@3.7.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
[question] have you setup an organizational secret called `GITHUB_TOKEN`?

Reply:
`GITHUB_TOKEN` is created automatically, right? 
https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Reply:
Oh, looks like it is! Wonderful